### ISSUE TITLE
package, rendition and media overlays vocabs aren't neatly future proof

### ISSUE THREAD

**Issue by mattgarrish** [w3c/epub-specs#608]
We currently define these vocabularies in the specification documents which allows them to be adjusted to the version of epub that they apply to, but if we pull them out and put them under the /vocab branch we won't be able to nicely manage inclusion/deprecation of properties over revisions.

We might be able to mitigate some of the problem by versioning the releases to match the epub version, but moving forward you'll have to know to click back in time to find the actual vocabulary you're allowed to use.

For example, if epub 3.2 adds a whole new set of fixed layout properties, these would then appear to be valid in 3.1 since they'll both reference the same package metadata vocab.

We also have a problem that the base IRIs for these use non-descript paths like http://www.idpf.org/epub/vocab/package
Although this is a mostly invisible to users, if you were to resolve the property to look it up you'll always end up at the latest version no matter which version of epub 3 you're creating. But any future dramatic change to epub that would want to define new package (or other) metadata would need to find a new base IRI in order not to collide with what we've already done for epub 3.

Long story short, I don't have a good solution to this. I guess we leave things as they are and hope for the best, which probably means managing through version number changes in the package + mods to epubcheck to handle variations.

Recording in case someone has a bright idea in the future.

/cc @mgylling @TzviyaSiegman

**Comment by mattgarrish** [comment]
Closing this issue. We want the vocabularies to live with epub 3 and beyond. If metadata gets added later that is incompatible with epub 3, we've already discussed that we would note such application in the vocabularies.
